google-ads-examples/src/main/java/com/google/ads/googleads/examples/reporting/ParallelReportDownload.java

```diff
@@ -242,12 +245,12 @@ boolean isSuccess() {
 
     @Override
     public String toString() {
-      return "Customer ID "
+      return "Customer ID '"
           + customerId
-          + " Number of responses: "
+          + "' Number of responses: "
```

Comment:
Why is there a single quote added around Customer ID and Number of responses here?